app/views/wpcc/your_contributions/new.html.erb

```diff
@@ -1,6 +1,6 @@
 <%= render 'wpcc/shared/your_details', message_presenter: @message_presenter %>
 
-<section class="contributions">
+<section class="section--contributions contributions">
```

Comment:
is this class required for the tests?

Reply:
sorry, that was supposed to be removed again as I changed the working code. The push last night failed to complete. 🙈  closed my laptop too soon